package.json

```diff
@@ -53,6 +53,7 @@
     "babel-preset-stage-1": "^6.1.18",
     "chai": "^3.3.0",
     "del": "^2.0.2",
+    "dirty-chai": "^1.2.2",
```

Comment:
Function form for terminating assertion properties. https://github.com/prodatakey/dirty-chai